components/youtube/new-videos.js

```diff
@@ -0,0 +1,52 @@
+const axios = require("axios");
```

Comment:
You can remove all instances of these requires, since you don't appear to use `axios` in these files. Instead you can just require it once in the `youtube` app file.